extension-manifest-v3/src/content_scripts/whotracksme/reporting.js

```diff
@@ -1,4 +1,4 @@
-import { recordMouseDown } from '@whotracksme/webextension-packages/packages/reporting/content-script';
+import { recordMouseDown } from '@whotracksme/webextension-packages/packages/reporting/src/request/content-script.js';
```

Comment:
There is an issue with using nested `package.json` with the newest rollup. I suggest flattening the `@whotracksme/webextension-packages` project to a single package (containing communication and reporting) - for now we also send a lot of garbage data to npm, like tests, examples, etc.. (they are not cleared when main package is created).

Reply:
I think we "hacked" a little, and they might change implementation of the module resolution in rollup, so the nested package.json stops working (beside the main import, as it still works). 

The error message when using exports:
```
[vite]: Rollup failed to resolve import "@whotracksme/webextension-packages/packages/reporting/content-script" from "/Users/dominik/Workspace/ghostery-extension/extension-manifest-v3/src/background/reporting/index.js".
This is most likely unintended because it can break your application at runtime.
```

Reply:
I checked, and it looks, that nested paths from a package are resolved to the path first, so there is no `reporting/content-script` and it fails. The `.../reporting` exists, and then it checks that there is `package.json`, so it uses it.

However, a package inside a package is not normally supported case, and we should flatten it. For now, we can import file directly, as I've changed it.